crowdin/src/main/java/com/crowdin/platform/util/Extensions.kt

```diff
@@ -17,6 +17,8 @@ import java.util.TimeZone
 
 const val NEW_LINE = "<br>"
 private const val DEFAULT_DATE_TIME_FORMAT = "yyyy_MM_dd-HH_mm_ss"
+private const val HEBREW_LANGUAGE_CODE = "iw"
```

Comment:
Let's try adding some mapping instead of separate constants. It will allow easily add of new languages if it'll be needed

Reply:
Upadted